python/cudf/cudf/core/index.py

```diff
@@ -867,6 +879,18 @@ def min(self):
     def max(self):
         return self._minmax("max")
 
+    @property
+    def values(self):
+        return cupy.arange(self.start, self.stop, self.step)
+
+    def any(self):
+        if len(self) == 0:
+            return False
+        return True
```

Comment:
Unfortunately this isn't quite sufficient. You can also have a `RangeIndex` that only contains 0!
```
>>> list(pd.RangeIndex(0, 1))
[0]
>>> pd.RangeIndex(0, 1).any()
False
```